tools/karma_template.conf.js

```diff
@@ -66,7 +69,28 @@ const CUSTOM_LAUNCHERS = {
     browser: 'chrome',
     browser_version: '104.0',
```

Comment:
Can we use a higher version? Chrome Stable should be M112 now, while Canary is M113.

Reply:
WebGPU tests are set to use the latest Chrome version available on BrowserStack. We can probably also raise the Chrome version we use for other tests. I think 104 is somewhat old.